http/swagger.yml

```diff
@@ -12036,12 +12036,10 @@ components:
           - retention_policy
     DBRPs:
       properties:
-        notificationEndpoints:
+        content:
```

Comment:
To match the convention of other schemas, I think this should be `dbrps`

Reply:
This is IMHO just a copy-paste error in the swagger file. The existing `influx v1 auth` command already uses the `content` property and it already works well in OSS and in the cloud, the client also uses https://github.com/influxdata/influxdb/blob/master/dbrp/http_server_dbrp.go#L128. IMHO it would be better to fix it to match the implementation state first and then possibly create a defect for a better name and links later as a second step if you want to, WDYT?

Reply:
Yeah, with fresh eyes it sounds like a good idea to fix for now and then improve later 👍